src/neighbours.jl

```diff
@@ -85,3 +86,65 @@ function find_neighbours!(s::Simulation,
         end
     end
 end
+
+
+"""
+    DistanceNeighbourFinderTree(nb_matrix, n_steps, dist_cutoff)
```

Comment:
Rename to `DistanceTreeNeighbourFinder` or similar? I kind of like the convention of finishing all neighbour finders with `NeighbourFinder`.

Reply:
Or just `TreeNeighbourFinder` maybe?

Reply:
Sounds good.